debian/ooniprobe-cli.service

```diff
@@ -9,6 +9,7 @@ ConditionPathExists=/etc/ooniprobe/ooniprobe.conf
 
 [Service]
 Type=simple
+ExecStartPre=-/bin/rm -rf /var/lib/ooniprobe/msmts/*
```

Comment:
There is no longer the need to do this as it's done automatically by probe-cli since version 3.2.0